python/tutorials/gluon/01-attention-forward.py

```diff
@@ -1067,7 +1023,10 @@ def bench(Z, H, N_CTX, HEAD_DIM, causal, provider):
         else:
             raise ValueError(f"Unsupported provider: {provider}")
 
-        ms = triton.testing.do_bench(fn)
+        if not profile:
```

Comment:
If `profile=False`, why invoking the `do_bench` function? Seems counter intuitive